src/AccessibilityInsights.Extensions.AzureDevOps/ConnectionInfo.cs

```diff
@@ -128,7 +127,7 @@ public void SetLastUsage(DateTime? updateTime = null)
             ConcreteLastUsage = newTime;
         }
 
-        public bool Equals(IConnectionInfo other)
+        public bool Equals(ConnectionInfo other)
```

Comment:
I forget--are we removing the definitions for  IConnectionInfo, ITeam, and IProject, since they're no longer used in the interface?

Reply:
Yes